misc/collect.py

```diff
@@ -41,6 +46,21 @@ def change_prefix(images, old_prefix, new_prefix):
       exit(1)
     paths.append(path)
 
+# do not write data if output file is never then input data
+if args.write_new_data_only and os.path.exists(args.output):
+  p1 = os.path.getmtime(args.output)
```

Comment:
I would rather call it `json_output_path` and `profiles_input_path` or similar instead of p1 and p2

Reply:
p1 and p2 are timestamps...

Reply:
Well `json_output_timestamp` then?

Reply:
`output_mtime` and `input_mtime`?

Reply:
@aparcar I made it a bit better, but not quite what you asked for... is it ok if I merge this?